app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -981,7 +980,7 @@ private void checkPermissions() {
 
         if(ActivityCompat.checkSelfPermission(this,
                 Manifest.permission.RECORD_AUDIO) == PackageManager.PERMISSION_GRANTED) {
-            PrefManager.putBoolean(Constant.MIC_INPUT, checkMicInput());
+            PrefManager.putBoolean(Constant.MIC_INPUT, checkMicInput() && PrefManager.getBoolean(Constant.MIC_INPUT, false));
```

Comment:
@amitiwary999 Pls do this change while adding the feature sending the settings to server. 
@chiragw15 I have added the line comment.